bugbug/bugzilla.py

```diff
@@ -1,483 +1,490 @@
-# -*- coding: utf-8 -*-
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this file,
-# You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
It seams that every single line in this file got modified (i.e., replacing the end of line from `\n` to `\n\r`).  Please fix that where applicable (some other files have the same problem).